classes/image_types_ostree.bbclass

```diff
@@ -131,27 +130,6 @@ IMAGE_CMD_ostree () {
 
     ln -sf ../var/usrlocal usr/local
 
-    if [ "${KERNEL_IMAGETYPE}" = "fitImage" ]; then
-        # this is a hack for ostree not to override init= in kernel cmdline -
```

Comment:
Did you check if this hack/workaround is also not needed when migrating to ostree-kernel? The empty initramfs when using fit is needed in order for ostree to assume that the system got a initramfs in place (part of fit, in this case).

Reply:
Not yet, only verified it on a arm machine with devicetrees and initramfs, will do that and update the comment, see "WIP" for now.

Reply:
Verified it works for both fitimage and uImage.